picard/ui/options/maintenance.py

```diff
@@ -0,0 +1,182 @@
+        "Settings that are found in the configuration that do not appear on any option "
+        "settings page will be listed below. If your configuration does not contain any "
+        "unused option settings, then the list will be empty.\n\n"
+        "Note that unused option settings could come from plugins that have been removed, "
+        "so please be careful to not remove settings that you may want to use later when "
+        "the plugin is reloaded.  Options belonging to plugins that are loaded but "
+        "currently disabled will not be listed for possible removal.\n\n"
+        "To remove one or more settings, select the settings to remove by checking the "
+        "box next to the settings, and then enable the removal by checking the \"Remove "
+        "selected options\" box. When you choose \"Make It So!\" to save your option "
+        "settings, the selected items will be removed."
+    )
+
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self.ui = Ui_MaintenanceOptionsPage()
+        self.ui.setupUi(self)
+        self.ui.description.setText(_(self.INSTRUCTIONS))
+        self.ui.tableWidget.setHorizontalHeaderLabels([_("Option"), _("Value")])
+        self.ui.select_all.stateChanged.connect(self.select_all_changed)
+        self.ui.enable_cleanup.stateChanged.connect(self.enable_cleanup_changed)
+
+    def load(self):
+        config = get_config()
+
+        # Setting options from all option pages and loaded plugins (including plugins currently disabled).
+        key_options = set(config.setting.as_dict())
+
+        # Combine all page and plugin settings with required options not appearing in option pages.
+        current_options = set([
+            # Include options that are required but are not entered directly from the options pages.
+            'file_renaming_scripts',
+            'selected_file_naming_script_id',
+            'log_verbosity',
+            # Items missed if TagsCompatibilityWaveOptionsPage does not register.
+            'remove_wave_riff_info',
+            'wave_riff_info_encoding',
+            'write_wave_riff_info',
+        ]).union(key_options)
+
+        # All setting options included in the INI file.
+        config.beginGroup("setting")
+        file_options = set(config.childKeys())
+        config.endGroup()
+
+        orphan_options = file_options.difference(current_options)
+
+        self.ui.description.setText(
+            _(self.INSTRUCTIONS)
+            + _("\n\nThe configuration file currently contains {0} option settings, {1} which are unused.").format(
+                len(file_options),
+                len(orphan_options)
+            )
+        )
+        self.ui.enable_cleanup.setChecked(False)
+        self.ui.tableWidget.clearContents()
+        self.ui.tableWidget.setRowCount(len(orphan_options))
+        for row, item in enumerate(sorted(orphan_options)):
+            tableitem = QtWidgets.QTableWidgetItem(item)
```

Comment:
As explained below, it's prolly safer to differentiable table cell label from its data:

```python
            tableitem = QtWidgets.QTableWidgetItem("label:" + option_name)  # as example
            tableitem.setData(QtCore.Qt.UserRole, option_name)
```

And change code below to use data instead of text:

```python
 to_remove.add(item.data(QtCore.Qt.UserRole))
```